### PR TITLE
docs: add bun to README

### PR DESCRIPTION
**Pull request by ndungtse**
As when initializing nuxt project with nuxt, bun is included in package managers to be chosen so this pull request update package managers commands including bun. This applies to all all script commands mentioned (install, dev, build and preview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ pnpm install
 
 # yarn
 yarn install
+
+#bun
```
Comment (danielroe) [inline README.md]:
```suggestion
# bun
```

File: README.md [hunk 2]
```diff
@@ -30,6 +33,9 @@ pnpm run dev
 
 # yarn
 yarn dev
+
+# bun
+bun dev
```
Comment (danielroe) [inline README.md]:
```suggestion
bun run dev
```

File: README.md [hunk 3]
```diff
@@ -58,6 +67,9 @@ pnpm run preview
 
 # yarn
 yarn preview
+
+# bun
+bun preview
```
Comment (danielroe) [inline README.md]:
```suggestion
bun run preview
```

**Review by danielroe — APPROVED** [review]

**Comment by Atinux** [comment]
Shall we also add it to nuxt.new ?

**Pull request merged**